<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslateWidget</class>
 <widget class="QWidget" name="TranslateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>妙妙翻译</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowseroutput"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonTranslate">
     <property name="text">
      <string>翻译</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonLanguage">
     <property name="text">
      <string>英文-&gt;中文</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="textEditInput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
